libraries/botbuilder-ai/src/qnaCardBuilder.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Activity, MessageFactory, CardAction, CardFactory } from 'botbuilder-core';
+import { QnAMakerResult } from './';
+
+/**
+ * Provides methods to create activities containing hero cards for showing active learning or multi-turn prompt options for the QnAMakerDialog.
+ */
+export class QnACardBuilder {
```

Comment:
>QnACardBuilder [](start = 13, length = 14)

This should probably be named QnACardFactory to be consistent with our other factory classes.

Reply:
@Stevenic I am inclined to agree - issue is we already have a class in C# with this name.  Where do we stand in terms of amending that one to match?  Arguably the only place it is likely ot be used right now is QnAMakerDialog, but it technically would be a breaking change. https://github.com/microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs
cc @cleemullins